### PR TITLE
Fix/async await

### PR DESCRIPTION
**Pull request by mjdavidson**

### CODE REVIEW

File: js/stores/CurrentLocation.js [hunk 1]
```diff
@@ -16,14 +16,16 @@ class CurrentLocation extends Events {
       hasGranted: false,
       initialSet: window.location.pathname.split('/')[1] !== 's',
     }
+  }
+
+  async componentDidMount() {
```
Comment (mjdavidson) [inline js/stores/CurrentLocation.js:21]: @consindo how do you feel about this stuff moving out of the constructor - i don't think it should be there anyway

File: js/views/root/Content.jsx [hunk 2]
```diff
@@ -12,22 +12,13 @@ class RootContent extends React.Component {
   }
 
   state = {
-    stations: StationStore.getData(),
+    stations: null,
```
Comment (mjdavidson) [inline js/views/root/Content.jsx:15]: i also changed this to do an initial update on CDM

File: js/views/station/Save.jsx [hunk 3]
```diff
@@ -48,6 +28,24 @@ class Save extends React.Component {
     match: PropTypes.object,
   }
 
+  async componentDidMount() {
```
Comment (mjdavidson) [inline js/views/station/Save.jsx:31]: another CDM

**Review by mjdavidson — COMMENTED** [review]
is server-static still used?

File: js/stores/CurrentLocation.js (hunk 1 above)
Comment (consindo) [reply js/stores/CurrentLocation.js:21]: agree

File: js/views/root/Content.jsx (hunk 2 above)
Comment (consindo) [reply js/views/root/Content.jsx:15]: okay, as long as it renders I guess 🤷‍♀

File: package.json [hunk 4]
```diff
@@ -37,34 +37,35 @@
     "babel-eslint": "^10.0.2",
     "babel-loader": "^8.0.6",
     "babel-plugin-react-native-web": "^0.11.4",
-    "babel-preset-airbnb": "^4.0.0",
+    "babel-preset-airbnb": "^4.0.1",
     "cross-env": "^5.2.0",
-    "eslint": "^6.0.0",
+    "eslint": "^5.16.0",
```
Comment (consindo) [inline package.json:42]: why did we go back to v5?

**Review by consindo — APPROVED** [review]

**Pull request merged**